### ISSUE TITLE
Status line is too narrow

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#16]
```
Status line is a bit narrow so persistent messages in its right corner (for
example "updated" with a checkmark icon) overlap its border line.
Screenshot attached.
```

Original issue reported on code.google.com by `isagal...@gmail.com` on 18 Dec 2009 at 8:31

Attachments:
- [pino-narrow-status-line.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-16/comment-0/pino-narrow-status-line.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 18 Dec 2009 at 9:46
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 99928b6526 revision.
```

Original comment by `tro...@gmail.com` on 19 Dec 2009 at 3:10
- Changed state: **Fixed**
